worlds/__init__.py

```diff
@@ -111,8 +112,25 @@ def load(self) -> bool:
     else:
         world_source.load()
 
+
 from .AutoWorld import AutoWorldRegister
 
+for world_source in world_sources:
+    if not world_source.is_zip:
+        # look for manifest
+        manifest = {}
+        for dirpath, dirnames, filenames in os.walk(world_source.resolved_path):
+            for file in filenames:
+                if file.endswith("archipelago.json"):
+                    manifest = json.load(open(os.path.join(dirpath, file), "r"))
+                    break
+            if manifest:
+                break
+        game = manifest.get("game")
+        if game in AutoWorldRegister.world_types:
+            AutoWorldRegister.world_types[game]._world_version = Version(*tuplize_version(manifest.get("world_version",
```

Comment:
Hmm, is there a good reason why we need to import AutoWorldRegister *after* loading all the worlds, and if no, can this not just be inside the `else` clause for the code block above?

Reply:
In fact, that's not even necessary - This could make use of the return value of `world_source`, right? Unless we want to support the concept of world-less apworlds (I have one of these actually... haha)

Reply:
Wait, Universal Tracker is a world-less .apworld, and we definitely don't wanna kill that. Ok, so ignore the second comment :D